text/052-platform-release-cycle.md

```diff
@@ -0,0 +1,174 @@
+built. Coq point release are then used to quickly deliver fixes
+to the platform.
+
+# Motivation
+
+The beta testing period does not provide enough feedback to sensibly improve
+Coq between the `VX+beta1` and `VX.0` tags. The reasons are, most important
+last:
+- users have better things to do, frankly
+- one month is short, especially with Christmas, summer holydays or conferences
+  around (our beta period is December or May)
+- in order to test Coq you are likely to need a bunch of Coq packages, and they
+  are typically only available after the beta period is over
+
+Moreover the "beta period" has also been seen by devs as a time frame where
+they can still change many things. As a results users are even less happy
+to spend time working on a moving target.
+
+Finally, due to the cost of backporting breaking changes from the master
+branch to the release branch, this only happens very early in the release
+cycle (for example only 2 such changes were backported in the beta period
+for 8.12, 0 for 8.13).
+
+## New actor: platform
+
+The Coq platform is a selection of Coq packages which work well
+for a given Coq version. One of its objective is to disentangle the release
+of Coq with the release of other packages. It provides scripts that build
+that selection of Coq packages on Windows, Mac and Linux from sources
+on the user's machine, and as well binary installers (prebuilt) for users
+to quickly get a working environment.
+
+**The platform is the final product users will get** in this new model;
+Coq is a core component of it.
+
+# Detailed design
+
+The new process' timeline:
+```
+                     DO+           O                         O   O  O
+coq      --+---(1)---+------(2)----+----------(3)------------+---+--+----
+ vX branch/          |             |                         |
+           VX+rc tag/              |                         |
+                          VX.0 tag/                          |
+                                                    VX.1 tag/
+   
+                                  DI                   DI      DI
+platform  -----------+----(4)------+--------(5)---------+-------+-------
+           vX branch/              |                    |       |
+                      VX.0+beta tag/                    |       |
+                                              VX.0.0 tag/    .../
+Artifacts:
+- D = docker image for Coq (or Coq + the platform)
+- O = OPAM package (O+ means for core-dev, otherwise it is for the main OPAM repo)
+- I = binary installers for Coq platform
+```
+
+## Coq
+
+On time based schedule the RM branches vX.
+
+1. The RM shepherds the few PR which are ready and pins projects tracked by CI
+   (using commit hashes, not necessarily tags).
+   This step should take no more than 2 weeks.
+   - The RM **tags** VX+rc, but produces no binary installers, see the platform release.
+   - The RM writes an **OPAM package**, currently the policy is to upload it to
```

Comment:
```suggestion
   - The RM writes or request help from opam users to write an **OPAM package**, currently the policy is to upload it to
```

BTW, I probably said that already, but opam is never branded in all caps in the official channels.